sdk/Lusid.Sdk.Tests/tutorials/Ibor/PortfolioCashFlows.cs

```diff
@@ -243,11 +243,11 @@ internal static List<TransactionRequest> MapToCashFlowTransactionRequest(IEnumer
         // Given a transaction, this method creates a TransactionRequest so that it can be upserted back into LUSID.
         // InstrumentUid is additionally added to identify where the cashflow came from. The transaction ID needs to
         // be unique.
-        internal static IEnumerable<Transaction> PopulateCashFlowTransactionWithUniqueIds(IEnumerable<Transaction> transactions, string cashFlowCurrency)
+        internal static IEnumerable<Transaction> PopulateCashFlowTransactionWithUniqueIds(IEnumerable<Transaction> transactions)
         {
             foreach (var transaction in transactions)
             {
-                transaction.InstrumentIdentifiers.Add("Instrument/default/Currency", cashFlowCurrency);
+                transaction.InstrumentIdentifiers.Add("Instrument/default/Currency", transaction.TransactionCurrency);
```

Comment:
refactor, don't see why we need a separate parameter specifying cashflow ccy when the transaction should know this info